apps/meteor/tests/end-to-end/api/teams.ts

```diff
@@ -172,6 +172,84 @@ describe('[Teams]', () => {
 				})
 				.end(done);
 		});
+
+		it('should create a team with sidepanel items containing channels', async () => {
+			const teamName = `test-team-with-sidepanel-${Date.now()}`;
+			const sidepanelItems = ['channels'];
+
+			const response = await request
+				.post(api('teams.create'))
+				.set(credentials)
+				.send({
+					name: teamName,
+					type: 0,
+					sidepanel: {
+						items: sidepanelItems,
+					},
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res) => {
+					expect(res.body).to.have.property('success', true);
+				});
+
+			await request
+				.get(api('channels.info'))
+				.set(credentials)
+				.query({ roomId: response.body.team.roomId })
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((response) => {
+					expect(response.body).to.have.property('success', true);
+					expect(response.body.channel).to.have.property('sidepanel');
+					expect(response.body.channel.sidepanel).to.have.property('items').that.is.an('array').to.have.deep.members(sidepanelItems);
+				});
+			await deleteTeam(credentials, teamName);
+		});
+
+		it('should throw error when creating a team with sidepanel with more than 2 items', async () => {
+			await request
+				.post(api('teams.create'))
+				.set(credentials)
+				.send({
+					name: `test-team-with-sidepanel-error-${Date.now()}`,
+					type: 0,
+					sidepanel: {
+						items: ['channels', 'discussion', 'other'],
+					},
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(400);
+		});
+
+		it('should throw error when creating a team with sidepanel with incorrect items', async () => {
+			await request
+				.post(api('teams.create'))
+				.set(credentials)
+				.send({
+					name: `test-team-with-sidepanel-error-${Date.now()}`,
+					type: 0,
+					sidepanel: {
+						items: ['other'],
+					},
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(400);
+		});
+		it('should throw error when creating a team with sidepanel with duplicated items', async () => {
+			await request
+				.post(api('teams.create'))
+				.set(credentials)
+				.send({
+					name: `test-team-with-sidepanel-error-${Date.now()}`,
+					type: 0,
+					sidepanel: {
+						items: ['other'],
```

Comment:
```suggestion
						items: ['channels', 'channels'],
```
Weren't the items here supposed to be duplicated? :thinking:

Reply:
yes indeed haha thanks!